src/dict/loaddictionaries.cc

```diff
@@ -94,7 +93,14 @@ void LoadDictionaries::run()
   try {
     for ( const auto & path : paths ) {
       qDebug() << "handle path:" << path.path;
-      handlePath( path );
+      try {
+        handlePath( path );
+      }
+      catch ( const std::exception & e ) {
+        qWarning() << "Error handling path:" << path.path << "-" << e.what();
+        //hold last exception message.
+        exceptionText = "[" + path.path.toStdString() + "]:" + e.what();
```

Comment:
`exceptionText` should be a String List to collect all errors instead of the last one only .

Reply:
done